gpii/node_modules/flowManager/test/CaptureTests.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * GPII Capture Tests
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = require("kettle");
+
+var jqUnit = require("node-jqunit");
+
+fluid.require("%gpii-universal");
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.flowManager.capture");
+
+kettle.config.createDefaults({
+    configName: "gpii.flowManager.tests.capture.fakeData.config",
+    configPath: "%gpii-universal/gpii/node_modules/flowManager/test/configs"
+});
+
+gpii.tests.flowManager.capture.checkForFakeMags = function (payload) {
+    jqUnit.assertDeepEq("FakeMag1 with multiple settings handlers", payload.fakemag1, {
+        "magnification": 2,
+        "invert": true
+    });
+
+    jqUnit.assertDeepEq("FakeMag2 with single settings handler", payload.fakemag2, {
+        "magnification": 2,
+        "invert": true
+    });
+};
+
+gpii.tests.flowManager.capture.platformReporter = function () {
+    return {
+        id: "darwin"
+    };
+};
+
+gpii.tests.flowManager.capture.checkForInstalledMags = function (payload) {
+    jqUnit.assertEquals("There should be 2 solutions installed", 2, Object.keys(payload).length);
+    jqUnit.assertEquals("Check for Fake Mag 1", payload.fakemag1.name, "Fake Magnifier 1");
```

Comment:
When using `jqUnit.assert`, I believe the second argument should be the expected value and the third argument is the value to be asserted.

Reply:
right on, updated all the assert statements across the files